worlds/bk_sudoku/docs/setup_fr.md

```diff
@@ -0,0 +1,24 @@
+# Guide de configuration de BK Sudoku
+
+## Logiciels requis
+- [Bk Sudoku](https://github.com/Jarno458/sudoku)
+- Windows 8 ou supérieur
+
+## Concept général
+
+Il s'agit d'un client qui peut se connecter à n'importe quel slot du multi-monde, et qui vous permet de jouer au Sudoku pour débloquer des indices aléatoires pour les emplacements de ce slot.
```

Comment:
Suggesting to change "slot" to "world"

Replace "au Sudoku" by "à Sudoku"

Reply:
I also suggest to drop "aléatoire" here and leave the explanation to the guide. If not, you could re-word as previous with something like "pour obtenir des hints, soit le nom de l'item se trouvant à une location de ce monde. Notez que la location est choisie aléatoirement parmi toutes les locations qui ne sont pas trouvées dans le monde que vous êtes connectées."